CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## 2022-11-01
+
+### Changes
+
+---
+
```

Comment:
Just talked about this and we decided it's not necessary.

Reply:
Deleted 👍